www/js/con-author.js

```diff
@@ -745,9 +745,6 @@ define([
 				m = new memory({data: units});
 				unitsWidget.set("store", m);
```

Comment:
Yes true, like discussed left additional comments just in case someone in future might want to explore the capabilities of memory object wrapper